src/Resources/config/services.yaml

```diff
@@ -14,3 +14,10 @@ services:
 
     Becklyn\RouteTreeBundle\PostProcessing\Processor\Security\SecurityChecker:
         $trustResolver: '@security.authentication.trust_resolver'
+
+    Becklyn\RouteTreeBundle\KnpMenu\Renderer\SimpleTwigRenderer:
+        tags:
+            - { name: knp_menu.renderer, alias: route_tree }
```

Comment:
Do we want to keep the duplicate hardcoding for the alias or do we want to use something like `!php/const:Becklyn\RouteTreeBundle\KnpMenu\Renderer\SimpleTwigRenderer::ALIAS`?

Reply:
Sure, why not.

btw: you are using the old syntax, the new one is: 

```yaml
key: !php/const Becklyn\RouteTreeBundle\KnpMenu\Renderer\SimpleTwigRenderer::ALIAS
```